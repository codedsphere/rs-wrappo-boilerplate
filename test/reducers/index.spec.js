import * as reducers from 'mt-reducers';

describe('reducers', () => {
  it('should have the correct reducers', () => {
    expect(reducers).toMatchSnapshot();
  });
});
