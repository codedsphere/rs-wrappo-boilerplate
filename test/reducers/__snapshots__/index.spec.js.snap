// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducers should have the correct reducers 1`] = `
Object {
  "default": Object {
    "pageModel": [Function],
    "userModel": [Function],
  },
}
`;
